### PR TITLE
Issue10 retrieve organizations

### PR DESCRIPTION
**Pull request by VootLejin**
This also has a basic solution in place for issue #11, but it is not pretty.

### CODE REVIEW

File: app/views/dashboard.pug [hunk 1]
```diff
@@ -24,6 +24,7 @@ html(lang="en")
                         li: a(href="#viewusers" data-toggle="tab") View Users
                         li: a(href="#users" data-toggle="tab") Create Users
                         li:  a(href="#org" data-toggle="tab") Organization
+                        li: a(href="#contacts" data-toggle="tab") Contacts
```
Comment (DannyPeck) [inline app/views/dashboard.pug]: This should be under the Organizations tab not contacts

File: app/views/dashboard.pug [hunk 2]
```diff
@@ -54,5 +55,10 @@ html(lang="en")
                                         div.form-group
                                             button.btn.btn-default(type="submit") Create Account
                         .tab-pane(id="org") Organization
+                        .tab-pane(id="contacts") Contacts
```
Comment (DannyPeck) [inline app/views/dashboard.pug]: Should be under organizations not contacts

File: app/controllers/OrganizationController.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        request
+            .get ('localhost:5000/v1/organizations')
+            .set  ('Authorization', 'bearer ' + token)
+            .end  (function (err, resp) {
+                if (err) {
+                    console.log (err);
+                    return res.render('dashboard.pug', {message: "Sorry! Unable to Fetch Organizations."})
+                } else {
+
+                    // Build list of Organizations
```
Comment (DannyPeck) [inline app/controllers/OrganizationController.js:24]: Try and see if we can just use the list of organizations that are returned.

File: app/views/organizationlist.pug [hunk 4]
```diff
@@ -0,0 +1,9 @@
+//Broady speaking We probably want the following information:
+    //Name, Industry
+div.container
```
Comment (DannyPeck) [inline app/views/organizationlist.pug]: We also want the table to be inside the tab and not on a new page.

**Review by DannyPeck — CHANGES_REQUESTED** [review]
Its looking good, but a few questions.

File: app/controllers/OrganizationController.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+                    var list = {};
+                    for (var x in resp.body.organizations){
+                        list[x] = resp.body.organizations[x].name;
+                    }
+                    console.log(list);
```
Comment (DannyPeck) [inline app/controllers/OrganizationController.js]: please remove this console.log statement

**Comment by DannyPeck** [comment]
This is also random, there is a console.log statement in the UserController, please remove it, it is printing the resp.body.

**Comment by VootLejin** [comment]
Updated the List to be under Organizations and moved stuff to a more object oriented approach for the errors and such.

**Pull request merged**